hotfixes/wc-api-dev-cheque-defaults.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Prevents the cheque/check payment method from being auto enabled if settings haven't been configured.
+ * It also sets the description/instructions defaults.
+ *
+ * @since 0.8.1
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_api_dev_cheque_defaults( $settings ) {
+	$settings['enabled']['default'] = 'no';
+	$settings['description']['default'] = __( 'Pay for this order by check.', 'wc-api-dev' );
```

Comment:
OK — I thought it would be something like that, but without a note I wasn't sure. Documenting in the readme sounds good 👍